requirements.txt

```diff
@@ -4,6 +4,7 @@ nose==1.3.3
 requests==2.10.0
 celery==4.1.1
 newrelic
+redis
```

Comment:
I am concerned if we don't pin this version, it will come and bite us when we do deployments in the future.